daemon/execdriver/native/create.go

```diff
@@ -212,6 +212,20 @@ func (d *driver) setupRlimits(container *configs.Config, c *execdriver.Command)
 }
 
 func (d *driver) setupMounts(container *configs.Config, c *execdriver.Command) error {
+	userMounts := make(map[string]bool)
```

Comment:
@icecrime @mrunalp It's more of a personal/team/project preference. The `map[Type]bool` is a bit cleaner code-wise, but the `map[Type]struct{}` uses less memory. For this reason, the Docker project tends toward the `map[Type]struct{}` idiom instead.